dev_setup.sh

```diff
@@ -93,7 +93,14 @@ install_deps() {
     if found_exe apt-get; then
         $SUDO apt-get install -y git python python-dev python-setuptools python-virtualenv python-gobject-dev virtualenvwrapper libtool libffi-dev libssl-dev autoconf automake bison swig libglib2.0-dev s3cmd portaudio19-dev mpg123 screen flac curl libicu-dev pkg-config automake libjpeg-dev libfann-dev build-essential jq
     elif found_exe pacman; then
-        $SUDO pacman -S --needed git python2 python2-pip python2-setuptools python2-virtualenv python2-gobject python-virtualenvwrapper libtool libffi openssl autoconf bison swig glib2 s3cmd portaudio mpg123 screen flac curl pkg-config icu automake libjpeg-turbo base-devel jq
+        $SUDO pacman -S --needed --noconfirm git python2 python2-pip python2-setuptools python2-virtualenv python2-gobject python-virtualenvwrapper libtool libffi openssl autoconf bison swig glib2 s3cmd portaudio mpg123 screen flac curl pkg-config icu automake libjpeg-turbo base-devel jq
+        pacman -Qs "^libfann$" &> /dev/null || (
```

Comment:
Got here via Google. It seems `libfann` isn't in the Arch repos any more. [`fann` in the AUR](https://aur.archlinux.org/packages/fann/) still provides the lib.

Reply:
https://lists.archlinux.org/pipermail/aur-requests/2018-March/022950.html
https://lists.archlinux.org/pipermail/aur-requests/2018-March/022971.html

Reply:
Thanks for the context @Ceda-EI.

Reply:
@ConorIA check #1534  #1535